bootstrap.ps1

```diff
@@ -90,16 +90,21 @@ If ($lastexitcode -notmatch 0) {
 
 
 # Install OBS Studio
-"[INFO] Downloading OBS ..."
-pushd thirdParty
-If (Test-Path OBS-Studio-18.0.1-Full.zip) {
-    Remove-Item OBS-Studio-18.0.1-Full.zip
+If (Test-Path "C:\Program Files (x86)\obs-studio\bin\32bit\obs32.exe") {
+    "[INFO] OBS had been installed."
+} Else {
+    "[INFO] Downloading OBS ..."
+    pushd thirdParty
+    If (Test-Path "OBS-Studio-18.0.1-Full.zip") {

```

Comment:
I don't think we have to remove it, the folder `thirdparty` is listed in `.gitignore`.